src/main/java/com/indeed/jiraactions/JiraActionsIndexBuilderCommandLine.java

```diff
@@ -85,6 +85,10 @@ private void initialize(String[] args) {
             final String iuploadUsername = config.getString("iupload.username");
             final String iuploadPassword = config.getString("iupload.password");
             final String indexName = config.getString("indexname");
+            final boolean buildJiraIssues = config.getBoolean("buildjiraissues");
+            final int jiraIssuesRange = config.getInt("jiraissuesrange");
+            final String[] deliveryLeadTimeFieldArray = config.getStringArray("deliveryleadtimefields");
+            final String deliveryLeadTimeFields = COMMA_JOINER.join(deliveryLeadTimeFieldArray);
```

Comment:
See my comments in ActionFactory. You're getting a string separated list of fields from a properties file, representing it as an Array, joining it into a comma separated String, then splitting it again.